docs/import-from-github.mdx

```diff
@@ -1,54 +1,80 @@
 ---
-title: Import from GitHub
-description: 'Import your machines from GitHub into Stately Studio.'
+title: Connect GitHub repo
+description: 'Sync your machines from GitHub with Stately Studio.'
 ---
 
-There are currently two ways to import your machines from GitHub into Stately Studio:
-
-1. [Import all machines from a GitHub repository](#import-all-machines-from-a-github-repository).
-2. [Import machines from a GitHub URL.](#import-machine-from-a-github-url)
+You can connect a GitHub repo to a new project in Stately Studio, keeping updates between GitHub and Stately Studio in sync. Connecting a GitHub repo allows you to import your existing machines from GitHub and push changes to your machines back to your repo as pull requests.
 
 :::studio
 
-Import from GitHub is a [premium feature](/studio-pro-plan.mdx) of Stately Studio.
-
-We offer a **free trial** on the [Stately Studio Pro account](https://stately.ai/pricing) so you can explore how our Pro features work for you and your team.
+Connect GitHub repo is a premium feature of Stately Studio. You can try Stately Studio’s premium plans with a free trial. [Check out the features on our Pro plan](studio-pro-plan.mdx), [Team plan](studio-team-plan.mdx), [Enterprise plan](studio-enterprise-plan.mdx) or [upgrade your existing plan](https://stately.ai/registry/billing).
 
 :::
 
-## Import all machines from a GitHub repository
+## Connect GitHub repo
 
-Use **Import from GitHub** to import all the machines from your GitHub repository into one project in the Stately Studio.
+There are three steps to setting up your GitHub repo as a connected project:
 
-:::info
+1. Select the repo. You can choose the repos based on your GitHub permissions.
+2. Select the branch. You can choose any branch in your repo.
+3. Choose the files you want to sync. You can sync any JavaScript or TypeScript files, but only XState machines will be imported.
 
-[Watch Anders demo Import from Github during our office hours](https://www.youtube.com/watch?v=k-ymwkyUBao&t=365s).
+### Auto-sync
 
-:::
+Enabling **auto-sync** will automatically fetch the latest changes from your connected repository every time you open or refresh the project in the Studio.
+
+Disable auto-sync from the GitHub options from the <Github size={18}/> repository name in the footer of the left drawer.
+
+### Path for new files
+
+You can choose a **Path for new files** for when you create new machines in this connected project. These machines will be added at this relative path from the root of your repo. The default path is **src/stately-studio** and will be created in your repo when you create your first pull request in the Studio.
```

Comment:
```suggestion
You can choose a **Path for new files** for when you create new machines in this connected project. These machines will be added at this relative path from the root of your repo. The default path is **src/stately-studio** and will be created in your repo the firs time you add new machines from Studio to a pull request.
```

@mellson Can you confirm that this is accurate, that it only adds this folder when needed? Otherwise, I can test it out tomorrow.

Reply:
Correct: It only adds this on demand. If the folder doesn't already exist, it will be added when the first pull request is created in the Studio.

Reply:
@laurakalbag I have a typo in my suggestion, "firs time". Sorry.